src/data/contributors/Rohit.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "Rohit",
+  "skills": ["Python", "c++","MYSQL",django],
```

Comment:
put django inside quotes, otherwise it will be interpreted as variable which is undefined.